lib/presentation/mixins/single_image_picker_mixin.dart

```diff
@@ -37,18 +38,20 @@ mixin SingleImagePickerMixin on CommonMediaPickerMixin {
         expandedWidget: const SizedBox(height: 50),
```

Comment:
LGTM overrall. Just add this to style class.
Linagora-design-flutter PR merged

Reply:
done